shoop/front/middleware.py

```diff
@@ -126,7 +126,11 @@ def process_view(self, request, view_func, *view_args, **view_kwargs):
             return maintenance_response
 
     def _get_maintenance_response(self, request, view_func):
-        if "login" in view_func.__name__ or getattr(view_func, "maintenance_mode_exempt", False):
+        if getattr(view_func, "maintenance_mode_exempt", False):
+            return
+        if "login" in view_func.__name__:
+            return
+        if getattr(request, "resolver_match") and getattr(request.resolver_match, "app_name") == "shoop_admin":
```

Comment:
Why not just `if request.resolver_match.app_name == "shoop_admin"`? (Since you're using the two-param getattr, i.e. without default value. That'll raise AttributeError if there is no such attribute.)